.github/workflows/test.yml

```diff
@@ -7,11 +7,6 @@ on:
       - 'rules/**'
       - 'tests/**'
       - '.github/**'
-  pull_request:
```

Comment:
Why did you remove this?

Reply:
Just trying to get it working. It worked once after doing chmod, but on push. On pull_request it failed to run the pipeline. 🤷